### ISSUE TITLE
error on checking duplication

### ISSUE THREAD

**Issue by kyungmi** [router5/router5#44]
Duplicated 'path' error was thrown when I tested the code below.

```javascript
new Router5()
    .addNode('users', '/users')          // #/users
    .addNode('groups', '/groups')        // #/groups
    .addNode('groups.users', '/users');  // #/groups/users
```

I think this logic `RouteNode.add()` for checking duplication may target the only nodes on the same level.
This is the error message.
```
Error: Path "/users" is already defined in route node
    at RouteNode.add (http://172.21.81.201:8081/node_modules/router5/dist/amd/router5.js:717:27)
    at RouteNode.addNode (http://172.21.81.201:8081/node_modules/router5/dist/amd/router5.js:766:22)
    at Router5.addNode (http://172.21.81.201:8081/node_modules/router5/dist/amd/router5.js:1103:31)
    at Router.start (http://172.21.81.201:8081/src/core/router.js:67:14)
    at http://172.21.81.201:8081/node_modules/async/dist/async.min.js:1:2339
    at P.forEachOf.P.eachOf (http://172.21.81.201:8081/node_modules/async/dist/async.min.js:1:5585)
    at L (http://172.21.81.201:8081/node_modules/async/dist/async.min.js:1:2319)
    at Object.P.parallel (http://172.21.81.201:8081/node_modules/async/dist/async.min.js:1:9044)
    at App._startPhase (http://172.21.81.201:8081/src/app.js:96:15)
    at r (http://172.21.81.201:8081/node_modules/async/dist/async.min.js:1:9224)
```

**Comment by troch** [comment]
Should be fixed in `2.0.3`, thanks for raising.

**Comment by kyungmi** [comment]
It is working well now. Thanks a lot! :)
